src/api-ms-win-core-synch-l1-2-0/api-ms-win-core-synch-l1-2-0.vcxproj

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup Label="Globals">
+    <VCProjectVersion>16.0</VCProjectVersion>
+    <Keyword>Win32Proj</Keyword>
+    <ProjectGuid>{9cf74355-f018-4c19-81ad-9dc6b7f2c6f5}</ProjectGuid>
+    <RootNamespace>apimswincoresynchl120</RootNamespace>
+    <ConfigurationType>DynamicLibrary</ConfigurationType>
+  </PropertyGroup>
+  <Import Project="$(SolutionDir)src\common.build.pre.props" />
+  <ItemGroup>
+    <ClCompile Include="main.cpp" />
+  </ItemGroup>
+  <ItemGroup>
+    <None Include="definitions.def" />
+  </ItemGroup>
+  <Import Project="$(SolutionDir)src\common.build.post.props" />
+  <ItemDefinitionGroup>
+    <ClCompile>
+      <PrecompiledHeader>NotUsing</PrecompiledHeader>
+    </ClCompile>
+    <Link>
+      <AdditionalDependencies>%(AdditionalDependencies)</AdditionalDependencies>

```

Comment:
This line is a no-op

Reply:
`common.build.post.props` adds a lot of libraries to the `AdditionalDependencies` by default for all projects.
This line removes those dependencies to ensure that no additional libraries are used accidentally.

Reply:
It doesn't, though. The inclusion of `%(AdditionalDependencies)` ensures that _all previous entries in the `AdditionalDependencies` metadata group are included_. :smile:

(this is rather like `int x = 5; x = x;`)

Reply:
I've changed it to `<AdditionalDependencies>kernel32.lib</AdditionalDependencies>`.
This should work right? The resolved dependencies view in the prop sheet indicates that `kernel32.lib` is now the only library.

Reply:
This should work :smile: